WeatherApp/Views/List/WeatherTableViewCell.swift

```diff
@@ -75,41 +78,44 @@ class WeatherTableViewCell: UITableViewCell {
     
     private func setupConstraints() {
         let iconDimension: CGFloat = 50
+        let iconOffset: CGFloat = 30
         let edgeMargin: CGFloat = 15
         let tempDistance: CGFloat = 5
         let labelsDistance: CGFloat = 20
         let cityLabelsWidth: CGFloat = 150
-
-        containerView.topAnchor.constraint(equalTo: self.contentView.topAnchor).isActive = true
-        containerView.bottomAnchor.constraint(equalTo:self.contentView.bottomAnchor).isActive = true
-        containerView.trailingAnchor.constraint(equalTo:self.contentView.trailingAnchor).isActive = true
-        containerView.leadingAnchor.constraint(equalTo: self.contentView.leadingAnchor).isActive = true
-        
-        weatherIcon.centerYAnchor.constraint(equalTo:self.contentView.centerYAnchor).isActive = true
-        weatherIcon.leadingAnchor.constraint(equalTo:self.contentView.leadingAnchor, constant: edgeMargin).isActive = true
-        weatherIcon.widthAnchor.constraint(equalToConstant: iconDimension).isActive = true
-        weatherIcon.heightAnchor.constraint(equalToConstant: iconDimension).isActive = true
         
-        cityLabel.centerYAnchor.constraint(equalTo:self.contentView.centerYAnchor).isActive = true
-        cityLabel.leadingAnchor.constraint(equalTo:self.weatherIcon.trailingAnchor, constant: labelsDistance).isActive = true
-        cityLabel.widthAnchor.constraint(equalToConstant: cityLabelsWidth).isActive = true
+        containerView.autoPinEdge(.top, to: .top, of: self.contentView)
```

Comment:
You could move all your `addSubview`() in this method, and then when you add one subview, you add all of its constraints underneath. For example:
```

contentView.addSubview(containerView)
containerView.autoPinEdge(.left, to: .left, of: self.contentView)
containerView.autoPinEdge(.right, to: .right, of: self.contentView)

contentView.addSubview(weatherIcon)
weatherIcon.autoPinEdgeToSuperViewEdge(..)
weatherIcon.autoPinEdgeToSuperViewEdge(..)
```
☝️  This is just an example with random constraints, not what you should really write. 

That way it's easier to see the view hierarchy because I don't have to scroll up to see which view belongs to which superview.